client/app/components/Input/InputPassword.jsx

```diff
@@ -0,0 +1,65 @@
+// @flow
+import React from 'react';
+import css from './Input.scss';
+
+export type Props = {
+  text?: string,
+  show?: boolean,
+  onChange?: Function,
+  onClick?: Function,
+};
+
+export type State = {
+  show?: boolean,
+};
+
+export class InputPassword extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { show: props.show };
+  }
+
+  ShowPassword = (show, text, onChange, onClick) => (
+    <div>
```

Comment:
We should also apply a `position: absolute` to this element so that the relative positioning of the show button is done in relation to this. 

Also what happens if someone's password takes up the full-length of the input field? The new show button would cover it. We'll need to account for that as well.